### ISSUE TITLE
Unable to set OpenVPN Client config protocol to TCP via UI

### ISSUE THREAD

**Issue by Codelica** [d3vilh/openvpn-ui#71]
Hello,

I'm trying out the stack here but seem to have run into a small bug. For some reason I'm unable to change the OpenVPN client protocol to TCP via the UI. 

After changing "Proto" to "tcp" and clicking the Save Config button, the client.conf always retains "udp".   If I manually edit the client.conf file to "tcp" I don't see the change updated in the UI form for Proto even after a container re-start/re-create.

Changing other parameters for for the client config seem to work and save properly (address, port, etc)

Thanks!

**Comment by d3vilh** [comment]
Hi James,
Indeed it looks like a bug, thanks for the reporting.

Could you let me know which openvpn-server and openvpn-ui images do you use and for which arch:
```bash
hiphopapapa@chapa:~ $ docker ps
CONTAINER ID   IMAGE                          COMMAND                  CREATED      STATUS        PORTS                    NAMES
cdcc47e4d7ea   d3vilh/openvpn-server:latest   "/bin/sh -c ./docker…"   6 days ago   Up 20 hours   0.0.0.0:1194->1194/udp   openvpn
572b4c4cd071   d3vilh/openvpn-ui:latest       "/bin/sh -c /opt/sta…"   6 days ago   Up 20 hours   0.0.0.0:8080->8080/tcp   openvpn-ui
hiphopapapa@chapa:~ $ docker inspect --format='{{json .Config.Labels}}' d3vilh/openvpn-ui:latest
{"maintainer":"Mr.Philipp <d3vilh@github.com>","version":"0.9.5.3"}
hiphopapapa@chapa:~ $ docker inspect --format='{{json .Config.Labels}}' d3vilh/openvpn-server:latest
{"maintainer":"Mr.Philipp <d3vilh@github.com>","version":"0.5.3"}
hiphopapapa@chapa:~ $ uname -a
Linux chapa 6.6.30-v8+ #1761 SMP PREEMPT Thu May  2 16:54:52 BST 2024 aarch64 GNU/Linux
```

Meanwhile I'll try to reproduce it on my `arm64` test system.

**Comment by Codelica** [comment]
Sure..  I'm running the "latest" tag for openvpn-server and openvpn-ui.  It's on a x86_64 install at Linode (Ubuntu 24).

**Comment by d3vilh** [comment]
Confirmed, as a bug, let me fix it.

**Comment by d3vilh** [comment]
@Codelica fixed in [`0.9.5.4`](https://github.com/d3vilh/openvpn-ui/releases/tag/0.9.5.4). 
All Docker hub images rebuild (except armv61, as Google did not build golang 1.22.3 images yet, for this legacy arch).
**Thank you for the reporting!**
